### PR TITLE
Setting a Canvas Width Default To Center Everything

### PR DESCRIPTION
**Pull request by tharmaman**

### CODE REVIEW

File: src/foam/graphics/TreeGraph.js [hunk 1]
```diff
@@ -329,7 +329,7 @@
         var delta = Math.abs(this.width - width) / width;
 
         if ( delta > 0.01 ) {
-          this.width = width;
+          this.width = width > 1600 ? width : 1600;
```
Comment (mcarcaso) [inline src/foam/graphics/TreeGraph.js]: `this.width = Math.min(width, 1600);`

**Review by mcarcaso — APPROVED** [review]

File: src/foam/graphics/TreeGraph.js (hunk 1 above)
Comment (kgrgreer) [reply src/foam/graphics/TreeGraph.js]: Why 1600?

Comment (tharmaman) [reply src/foam/graphics/TreeGraph.js]: Just changed it to use the clientWidth of the canvasContainer instead which makes more sense than the hard coded 1600.

Comment (tharmaman) [reply src/foam/graphics/TreeGraph.js]: Also added in mike's suggestion of Math.min

**Pull request closed without merging**